notebooks/a00_introduction_to_geomstats.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>NIT: geodescics -&gt; geodesics (at several places in the notebook)</p><p> </p><p> </p><p>NIT: geodesics in 2D and 3D space are straight lines. (The path *is* the geodesics, it does not follow a geodesic).</p><p>One path is a geodesic, and the other one is not a geodesic.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>done</p>